config/locales/ja.yml

```
@@ -226,10 +226,19 @@ ja:
       sign_up: "新規登録"
       forgot_password: "パスワードをお忘れですか?"
       resend_confirmation: "確認メールを再送しますか?"
+      continue_with_google: "Googleで続ける"
+      continue_with_apple: "Appleで続ける"
 
   profiles:
     show:
       settings: "設定"
+
+  users:
```

YAMLlint (1.38.0)

[error] duplication of key "users" in mapping

(key-duplicates)